inc/link.class.php

```diff
@@ -495,9 +495,10 @@ public function getSpecificMassiveActions($checkitem = null)
         $actions = parent::getSpecificMassiveActions($checkitem);
         $sep     = __CLASS__ . MassiveAction::CLASS_ACTION_SEPARATOR;
 
-        $actions[$sep . 'generation'] = __("Generate item", "order");
-        $actions[$sep . 'createLink'] = __("Link to an existing item", "order");
-        $actions[$sep . 'deleteLink'] = __("Delete item link", "order");
+        $actions[$sep . 'generation']       = __("Generate item", "order");
+        $actions[$sep . 'createLink']       = __("Link to an existing item", "order");
+        $actions[$sep . 'deleteLink']       = __("Delete item link", "order");
+        $actions[$sep . 'cancelReceipt']    = __("Cancel receipt", "order");
```

Comment:
Maybe "Cancel reception" ???
```suggestion
        $actions[$sep . 'cancelReceipt']    = __("Cancel reception", "order");
```